src/components/common/buttons.js

```diff
@@ -17,6 +17,7 @@ const buttonStyle = {
 };
 
 export const ButtonPrimary = ({ disabled, danger = false, children, ...props }) => {
+  const { colors } = useThemeFromContext();
```

Comment:
I was happy with `useTheme`... 😁

This was what I remembered from yesterday's meeting. I don't have a strong opinion on `useThemeContext` vs `useThemeFromContext`.

Reply:
If the theme is coming from the context, I prefer `useThemeFromContext` because it is clearer. `useThemeContext` sounds like the context is coming from the theme.

Reply:
Thinking about it some more, I would also vote for `useThemeFromContext`. Since our issue with `useTheme` was that it doesn't adequately highlight the dependency on a Context provider, I think `useThemeFromContext` highlights that dependency somewhat more clearly than `useThemeContext`.

Reply:
Good reasoning.   useThemeFromContext sounds good then.